editor/plugins/script_editor_plugin.cpp

```diff
@@ -1196,6 +1196,17 @@ void ScriptEditor::_menu_option(int p_option) {
 				editor->save_resource_as(current->get_edited_resource());
 
 			} break;
+			case FILE_OPEN_EXTERNAL: {
+				ScriptTextEditor *editor = Object::cast_to<ScriptTextEditor>(_get_current_editor());
+				if (!editor) return;
```

Comment:
No need to cast or null check here as its already casted and checked above to "current".

Reply:
Just this small change, then it's all good.